Framework/PythonInterface/mantid/_testing/AssertAlmostEqual.py

```diff
@@ -29,19 +29,24 @@ def assert_almost_equal(Workspace1, Workspace2, rtol=0.0, atol=0.0):
         If atol and rtol are both provided
 
     """
-
-    if rtol != 0.0 and atol != 0.0:
-        raise ValueError("Specify rtol or atol, not both")
+    rtol = kwargs.pop("rtol", None)
+    atol = kwargs.pop("atol", None)
     _rel = False
     tolerance = 1e-10
-    if atol:
-        tolerance = atol
 
-    if rtol:
-        tolerance = rtol
-        _rel = True
-
-    result, message = CompareWorkspaces(Workspace1, Workspace2, Tolerance=tolerance, ToleranceRelErr=_rel)
+    if rtol is not None and atol is not None:
```

Comment:
The story requested that if both were specified, run the algorithm twice, once with absolute tolerance, and once with relative tolerance.

Reply:
There was some contradiction between what is written in the story and the following discussion on slack channel. Ok, I am following requirements in the story.